### ISSUE TITLE
Spotify Family Account - Switching internally

### ISSUE THREAD

**Issue by chroth7** [vdesabou/alfred-spotify-mini-player#428]
Let me start with: Great workflow!

I have a small issue though:

We have a "Premium Family" account here, that includes several premium accounts. Within Spotify, I only have one "account/mail" to log into the Family Account, i.e. just one username.

When I set up an app in spotify it automatically targets the first "sub-account" in the group (family) and I don't see a way to change it on Spotify's app creation/edit/settings pages.

Is there a way to target a specific account in the family? 
Maybe I am blind and just didn't see the proper option

**Comment by vdesabou** [comment]
Thanks !

My understanding is that in a Family Plan, every user should have his own username/password (this is what I have in my case). I don't understand how you can only have only one username ? Can you double check that ?

If everyone has his own account, then you can use this [feature](https://alfred-spotify-mini-player.com/articles/multiple-users/).

**Comment by chroth7** [comment]
thanks for the quick feedback!

so you are right, I have indeed different logins, but what threw me off is the following:
- the browser remembers the login so it kept defaulting back to my primary account (which is not an issue), but:
- if you click "View Account" in the spotify webapp, it simply goes to the family accounts page without complaining and just shows you the currently logged in account in the browser (so say I am on account 2 in the app and click "View Account", I see account 1 in the web page)
- ... so I started using "private" browser windows, which kinda helped
- ... also the spotify dev page, they do NOT indicate which account in the family you are on, you don't see your account at all (at least I didn't)

... so that is all nice and weird, but well

I then went to create the apps, making sure being logged in as account2, and it still did NOT work.
After some more trying out other combinations I found a solution that works in my case:

- it seems to me that the app I create is NOT hardlinked to the account (1 or 2)...
- but the moment I authenticate against spotify (see screenshot of your instructions below), I have an option to change the account - and there was my problem: in this non-private/incognito window, I was still logged in as account 1. Once I changed THAT to account 2, it worked

![image](https://user-images.githubusercontent.com/6230368/99506888-fd793b00-2982-11eb-9949-1ebb5a9585a3.png)

Do I make sense?

Long story short: your app/workflow is doing well, but the spotify way of handling the accounts really confused me...

**Comment by vdesabou** [comment]
Cool, thanks for the feedback. Closing the ticket
